### PR TITLE
Feature/rate limit

### PR DESCRIPTION
**Pull request by melvinkoitzsch0007**
Rate Limit for ZAP Scans #20 (https://github.com/secureCodeBox/scanner-webapplication-zap/issues/20)

### CODE REVIEW

**Comment by J12934** [comment]
@Trosky we'll need to skip the signing setup in (external) pull requests, as they wont have access to the environment variables.

**Comment by rseedorff** [comment]
Hi  @melvinkoitzsch0007, thanks for your first contribution! ;)
Correct me if im wrong, but i think it would be better to try to merge this PR in the develop branch and not directly into the master.

**Comment by melvinkoitzsch0007** [comment]
Heyy @rseedorff yes, you're absolutely right. I will open a new pull request in the dev branch.

**Comment by J12934** [comment]
You can change the base branch of the pr via the webui without opening a new or.

File: scb-scanprocesses/zap-process/src/main/resources/forms/zap/configure-scanner-details.html [hunk 1]
```diff
@@ -17,6 +17,7 @@
     	});
     	camForm.on('submit', function () {
     	    $scope.targetList = $scope.targetList.map(function (target) {
+				<!-- target.attributes.ZAP_SCANNER_DELAY_IN_MS = splitOnNewline(target.attributes.ZAP_SCANNER_DELAY_IN_MS); -->
```
Comment (bitofant) [inline scb-scanprocesses/zap-process/src/main/resources/forms/zap/configure-scanner-details.html]: Either this is an intended comment (to be removed) or an equivalent line for the other property (thread count per host) is missing

**Pull request merged**